find-route/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.esri.arcgisruntime.sample.findroute">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:supportsRtl="true"
+        android:theme="@style/AppTheme">
+        <activity android:name="com.esri.arcgisruntime.sample.findroute.MainActivity">
```

Comment:
Add the following `android:configChanges="orientation|screenSize|uiMode">` to ensure the activity does not restart when screen changes.